READMEs/predictoor-dashboard.md

```diff
@@ -32,20 +38,25 @@ This command will open a browser window where you can select predictors and feed
 
 ## **How to use**
 
-By default, the plots will be empty because no predictors or feeds are selected.
+By default, if ppss is not configured with predictoor addresses, the plots will be empty because no predictors or feeds are selected. Otherwise you will see stats on the provided predictoor addresses.
```

Comment:
I suggest, "will be empty at start-up" to clarify that you can still select predictoors and feeds
also, I think this like is missing an "o" at "no predictors"